executor.c

```diff
@@ -13,16 +13,12 @@ int executor(char **args, char **env)
 	int status;
 	char *newCommand;
 
-	/* handeling exit */
-	if (_strcmp(args[0], "exit"))
+	/* handeling built-ins */
+	if (_strcmp(args[0], "exit") || _strcmp(args[0], "env"))
```

Comment:
`handle_builtins` already checks for `exit` and `env`, u just need to return 1 or 0 from `handle_builtns`